### PR TITLE
Fix misplaced '@' in RTL post meta.

### PR DESCRIPTION
**Pull request by jgarplind**
Addresses https://github.com/bluesky-social/social-app/issues/4529

The '@' symbol was placed at the right-side of the user handle, but the correct placement is always at the left-side since the user handle is inherently LTR.

Uses techniques from the specification: https://www.unicode.org/reports/tr9/

## Detailed problem description

`PostMeta.tsx` combines two separate text-nodes (handle + display name) into one wrapping text node. The BIDI algorithm (see link above) interprets the combined text as mixed LTR and RTL and applies logic as per the specification. As part of this logic, the `@` sign is placed on the right side (i.e. "before" in RTL terms) of the user's handle, since `@` is a "neutral character", and the combined text is apparently classified as RTL due to the RTL nature of the initial characters of the text (the display name).

By telling the algorithm to treat the handle (including the `@` character) as LTR text, this problem is mitigated.

## Show and tell

Before:
image

After:
image

🗒️ Note: Initially, the "After" was still justified towards the profile picture. Upon forking this repo and setting things up again, it was suddenly justified towards the right despite identical code. Now, after rebasing on top of @haileyok's changes, it looks fine again. Hence, I suspect my local environment rather than the code change to have temporarily caused the post meta to be justified to the right.

## Testing
✅ Tested on Windows Edge (as shown above).

❌ Don't have the necessary equipment at hand to test on iOS.

❌ Ran into setup problems when trying Android emulator, and didn't spend time trying to set up a Development build.

in other words, it would be helpful to have someone with a fully functioning setup give throw a glance.

### CODE REVIEW

**Review by haileyok — APPROVED** [review]
Nice! Thanks for digging at that. This knocks out one of the more annoying problems with LTR, appreciate you looking into it.
